mgmtsystem_nonconformity_claim/models/mgmtsystem_nonconformity_type.py

```diff
@@ -35,12 +35,3 @@ class mgmtsystem_nonconformity_type(orm.Model):
     _defaults = {
         'active': True,
     }
-
-
-class mgmtsystem_nonconformity(orm.Model):
-    _name = "mgmtsystem.nonconformity"
-    _inherit = "mgmtsystem.nonconformity"
-    _description = "Feedback and Nonconformities"
-    _columns = {
-        'type_id': fields.many2one('mgmtsystem.nonconformity.type', 'Type'),
-    }
```

Comment:
Not removed, just split.

See https://github.com/OCA/management-system/pull/33/files#diff-13471760dcc9d006b4e0e2f47672bfc4R24